dev/dags/dbt/jaffle_shop/profiles.yml

```diff
@@ -1,13 +1,12 @@
-# WARNING: this gets ignored by cosmos
 jaffle_shop:
   target: dev
   outputs:
     dev:
       type: postgres
-      host: localhost
-      user: postgres
-      password: postgres
+      host: postgres
+      user: airflow
+      password: pg_password
       port: 5432
-      dbname: public
-      schema: dbt_example
+      dbname: airflow
+      schema: public
```

Comment:
Let's use environment variables so each developer can configure this in the way it makes sense for them, and it would work for local development and remote.